### PR TITLE
Create wftpl_iam_-_groups_management_template.yaml

### PR DESCRIPTION
**Pull request by imrankhan4z**
Added a sample workflow to add IAM groups.

### CODE REVIEW

**Review by grabnerandi — CHANGES_REQUESTED** [review]
Hi. Can you also add a readme that explains workflow? Have a look at some of the other examples in the samples folder. all of them have a readme

**Comment by imrankhan4z** [comment]
Added the readme file. Please check.

**Review by grabnerandi — APPROVED** [review]

**Comment by grabnerandi** [comment]
Need one more approval for this review from the maintainers.
For reference - here the link to the LinkedIn Post from Imran who triggered this addition: https://www.linkedin.com/posts/imrankhan4z_dynatrace-saas-workflow-activity-7232411165534953483-7qnt?utm_source=share&utm_medium=member_desktop

**Review by miigwi — APPROVED** [review]

**Pull request merged**
